checkstyle/import-control.xml

```diff
@@ -74,6 +74,7 @@
     <subpackage name="protocol">
       <allow pkg="org.apache.kafka.common.errors" />
       <allow pkg="org.apache.kafka.common.protocol.types" />
+      <allow pkg="org.apache.kafka.clients.consumer" />
```

Comment:
Yes, this was a compatibility thing. I knew I couldn't move OffsetOutOfRangeException, so I allowed the import to use it within Errors. Part of what makes it challenging is its inheritance from InvalidOffsetException which is shared by NoOffsetForPartitionException too. I will see If I can do something better, or perhaps leave this exact exception out of the Errors mapping and fix this in another change. Especially since it relates to the larger discussion above.